fix py3-gcovr

smoser opened the pull request:
*  py3-gcovr - fix missing dep, add tests
 *  Add py/pip-check pipeline for running 'pip check'
    
    This is most useful for packages that we have manually
    encoded runtime package dependencies.  Running 'pip check'
    will verify that the versions we have match expectation.



Then... drop py/pip-check.

smoser committed:
Add py/pip-check pipeline for running 'pip check'

This is most useful for packages that we have manually
encoded runtime package dependencies.  Running 'pip check'
will verify that the versions we have match expectation.

smoser committed:
py3-gcovr - fix missing dep, add tests

pnasrat commented on pipelines/py/pip-check.yaml:
```diff
@@ -0,0 +1,42 @@
+name: Use pip from a zipapp for bootstrap. Only for use with bootstrapping.
```
please make the name and comments meaningful for what this pipeline does.

smoser replied:
done.

smoser committed:
Drop pip-check pipeline.

Discussed with pnasrat. There are benefits to running pip check,
but we'd like to do that more centrally (melange SCA?) or
correctly determine / generate runtime deps.

pnasrat reviewed (approved):
LGTM
